modules/cram/templates/sniffles2_single_call.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
+
+create_bed () {
+  echo -e "!{bedContent}" > "!{bed}"
+}
+
+# workaround for https://github.com/fritzsedlazeck/Sniffles/issues/373
+create_cram_slice () {
```

Comment:
see https://github.com/molgenis/vip/pull/339: depending on PR merge order a create_cram_slice _cleanup should be added

Reply:
discussed: will do after this PR